File: BEE 1094.java
import java.io.IOException;
import java.util.Scanner;
import java.text.DecimalFormat;

public class Main{

    public static void main(String[] args)  throws IOException{
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int c = 0, r = 0, s = 0, total = 0;
        for (int i = 0; i < n; i++) {
            int a = input.nextInt();
            String b = input.next();
            if (a >= 1 && a <= 15) {
                if (b.equalsIgnoreCase("C")) {
                    c += a;
                }
                if (b.equalsIgnoreCase("R")) {
                    r += a;
                }
                if (b.equalsIgnoreCase("S")) {
                    s += a;
                }
                total += a;
            }
        }

        System.out.println("Total: " + total + " cobaias");
        System.out.println("Total de coelhos: " + c);
        System.out.println("Total de ratos: " + r);
        System.out.println("Total de sapos: " + s);

        System.out.printf("Percentual de coelhos: %.2f %%\n", ((c * 100.0) / total));
        System.out.printf("Percentual de ratos: %.2f %%\n", ((r * 100.0) / total));
        System.out.printf("Percentual de sapos: %.2f %%\n", ((s * 100.0) / total));

    }
}

File: BEE 1094.cpp
#include<bits/stdc++.h>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(2) << (x)

using namespace std;

int main(){
    int n;
    cin >> n;
    int total = 0;
    int c = 0;
    int r = 0;
    int s = 0; 
    for (int i = 0; i < n; i++){
        int amount;
        char name;
        cin >> amount >> name;
        if (name == 'C'){
            c += amount;
        }
        else if (name == 'R'){
            r += amount;
        }
        else{
            s += amount;
        }
        total += amount;
    }
    cout << "Total: " << total << " cobaias" << endl;
    cout << "Total de coelhos: " << c << endl;
    cout << "Total de ratos: " << r << endl;
    cout << "Total de sapos: " << s << endl;
    cout << "Percentual de coelhos: " << FIXED_FLOAT(((1.0 * c) / total) * 100) << " %" << endl;
    cout << "Percentual de ratos: " << FIXED_FLOAT(((1.0 * r) / total) * 100) << " %" << endl;
    cout << "Percentual de sapos: " << FIXED_FLOAT(((1.0 * s) / total) * 100) << " %" << endl;
    return 0;
}